instructure/canvas-ios
# Add pronouns to Inbox

refs: MBL-13762
affects: student, teacher
release note: none

Test plan:
* Pronouns should show in:
  - Conversations list
  - Conversation details
  - Inbox compose
  - Address Book
* Log in as `s` on narmstrong
* Inbox > 2nd conversation in the list should display pronouns for Student One
* Sent > Pronouns test should display pronouns for Student One recipient
* Tap + to create a new conversation
* Select Course > CS 1400
* Tap + to add a person
* Search for "Student One"
* Pronouns should be displayed in result
* Tap Student One
* Pronouns should show in recipient token
